hearth/lib/hearth/cbor.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative 'cbor/half'
+require_relative 'cbor/tagged'
+require_relative 'cbor/encoder'
+require_relative 'cbor/decoder'
+
+module Hearth
+  # Hearth::Cbor is a purpose-built set of utilities for working with
+  # Cbor. It does not support all features of generic Cbor
+  # parsing and serialization.
+  # @api private
+  module Cbor
+    # Generic Cbor error, super class for specific encode/decode related errors.
+    class CborError < StandardError; end
```

Comment:
Ditto on CBOR for nested classes.

Reply:
Changed this to just `Error` since it will be `CBOR::Error`